### PR TITLE
Fixes #33453 - Remove pulp2 content classes

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
The single commit check seems to be busted here... merging!

**Comment by jlsherrill** [comment]
oh wait, hah no review yet

**Review by jjeffers — APPROVED** [review]
ACK

File: app/models/katello/deb.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Katello
   class Deb < Katello::Model
     include Concerns::PulpDatabaseUnit
 
-    CONTENT_TYPE = Pulp::Deb::CONTENT_TYPE
+    CONTENT_TYPE = 'deb'.freeze
```
[the remark this answers is not in the document]
Comment (jlsherrill) [reply app/models/katello/deb.rb]: yes, the pulp2 classes will go away, so we've been moving the type definition into the katello code.  We had already done this with most things like rpm, erratum, etc...

It looks like we did this in most cases when doing pulp3-required refactoring

[the remark this answers is not in the document]
Comment (jlsherrill) [reply app/models/katello/deb.rb]: i'm not sure what you mean by 'Why not continue to use a type?'.  Did that answer your question?

**Pull request merged**